README.md

```diff
@@ -160,35 +169,24 @@ Test all the things! We have a number of test suites, but are always looking to
 We recommend you read this article to understand better how we run our pipelines: https://www.jacobbolda.com/setting-up-ci-and-cd-for-tauri/
 
 ## Organization
-Tauri aims to be a sustainable collective based on principles that guide [sustainable
-free and open software communities](https://sfosc.org). You can get involved in many ways.
-
-This has been done with our best attempt at due diligence and in
-respect of the original authors. Thankyou - this project would never have
-been possible without your amazing contribution to open-source and we are
-honoured to carry the torch further. Of special note:
-- [zserge](https://github.com/zserge) for the original webview approach and
-go bindings
-- [Burtonago](https://github.com/burtonageo) for the Cargo Bundle prototype
+Tauri aims to be a sustainable collective based on principles that guide [sustainable free and open software communities](https://sfosc.org). To this end it has become a Programme within the [Commons Conservancy](https://commonsconservancy.org/), and you can contribute financially via [Open Collective](https://opencollective.com/tauri).
 
 ## Contributing
 Please make sure to read the [Contributing Guide](./.github/CONTRIBUTING.md)
 before making a pull request.
 
 Thank you to all the people who already contributed to Tauri!
 
-Special thanks to the development team at Volentix Labs for the encouragement and support in the early phases of Tauri, notably Rhys Parry and Gregory Luneau.
-
```

Comment:
Backtracking a bit, but two points:
- Have the removed special thanks become obsolete?
- Technically I think we're in the "setup phase" https://dracc.commonsconservancy.org/0020/ before formally going into effect as a programme. May be running a bit ahead of the actual status there.